test/lib/deployment/fetch-retry.js

```diff
@@ -33,7 +59,7 @@ async function fetchRetry(url, ...rest) {
         throw error;
       }
     },
-    { factor: 2, retries: 3 }
+    { factor: 2, retries: 5 }
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Why did we increase the general retry count? This won't impact the root problem, right?

Reply:
Oh this was left-over from previous retry handling we can revert it now